sklearn/model_selection/_search.py

```diff
@@ -767,8 +767,13 @@ def fit(self, X, y=None, *, groups=None, **fit_params):
             train/test set. Only used in conjunction with a "Group" :term:`cv`
             instance (e.g., :class:`~sklearn.model_selection.GroupKFold`).
 
-        **fit_params : dict of str -> object
-            Parameters passed to the ``fit`` method of the estimator.
+        **fit_params : dict of string -> object
```

Comment:
```suggestion
        **fit_params : dict of str -> object
```

We have a [Documentation Guideline](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation) which details how to document types. Item one:

> Use Python basic types. (bool instead of boolean)